### DISCUSSION TITLE
How to fix "ITimer1 was not declared in this scope" error

### DISCUSSION THREAD

**Discussion by Vonderhecke** [khoih-prog/TimerInterrupt#28] [General]
It wasn't a big deal to figure out why I got the error message "ITimer1 was not declared in this scope" even though the line "#define USE_TIMER_1 true" was present. But a small hint in the readme.md that "#include TimerInterrupt.h" may only appear after the define might be helpful.
Thanks for the nice lib, best regards, Ingo

**Comment by khoih-prog** [comment]
Hi @Vonderhecke 

Thank for the note which will help **inquisitive** users in avoiding the same error.

As you know, it's always better to start from the examples, then moving on after being familiarized with the library.

No instruction in README.md can cover every scenario. But I'll definitely try to add a note, if having chance.

I also change the Discussion topic, so that it's more obvious to other users.

Regards,
